Assets/Vimeo/Scripts/Recorder/Input/VideoInput.cs

```diff
@@ -72,12 +71,10 @@ public virtual void BeginRecording()
                 m_ModifiedResolution = true;
                 GameViewSize.SelectSize(size);
             }
-#endif            
         }
 
         public virtual void EndRecording()
         {
-#if UNITY_EDITOR
             if (m_ModifiedResolution)
             {
```

Comment:
I know it's not a part of this commit, but could be inlined -> `if (m_ModifiedResolution) {`